### PR TITLE
Initial Next.js frontend

### PR DESCRIPTION
**Pull request by dcramer**
This adds a POC for a Next.js frontend, converting all non-admin routes over.

There's a variety of improvements that will need backported to Remix if we dont move forward with this, such as the country page, the entities lists, etc.

### CODE REVIEW

**Comment by dcramer** [comment]
Going to merge the RPC changes into main as they're easy to pull out of this.

**Comment by dcramer** [comment]
I will note that I didn't try to just make existing React components compatible (which are in the `web` app), and instead chose to copy them. Creates a clean split, but its less code changed than it looks. That said a lot of the components _did_ need changes due to react-router bundler or 'use client' directives.

File: apps/web-next/src/app/%5Fhealth/route.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { type NextRequest } from "next/server";
```
Comment (r34son) [inline apps/web-next/src/app/%5Fhealth/route.tsx:1]: Is %5F intentional in path?

Comment (r34son) [reply apps/web-next/src/app/%5Fhealth/route.tsx:1]: image
Maybe it's smth with encoding in mobile gh app

File: apps/web-next/package.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@fontsource/raleway": "^5.0.19",
+    "@headlessui/react": "^1.7.19",
+    "@heroicons/react": "^2.1.3",
+    "@hookform/resolvers": "^3.6.0",
+    "@peated/design": "workspace:*",
+    "@peated/server": "workspace:*",
+    "@peated/tsconfig": "workspace:*",
```
Comment (r34son) [inline apps/web-next/package.json:17]: dev as well

File: apps/web-next/src/app/%5Fhealth/route.tsx (hunk 1 above)
Comment (dcramer) [reply apps/web-next/src/app/%5Fhealth/route.tsx:1]: Intentional - only way to escape an _ in path according to docs. Eg the path is ‘_health’

**Pull request merged**